category-press-news.php

```diff
@@ -0,0 +1,45 @@
+<?php get_header(); ?> 
+	
+	<!-- main content -->
+
+	<section id="main-content" class="two-col">
+
+  <?php
+  if( have_posts() ) {
+    while( have_posts() ) {
+      the_post();
+  ?>
+
+    <section class="main-col posts">
```

Comment:
Is this a typo? This is only supposed to exist once with all the posts inside it right? This should be outside the loop?